pages/models.py

```diff
@@ -277,8 +280,15 @@ def get_related_to(self, tag):
             from_element__tag=tag,
             from_element__dest_element=self)
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.__original_text = self.text
+
     def save(self, force_insert=False, force_update=False,
              using=None, update_fields=None):
+        if self.__original_text != self.text:
```

Comment:
That's an interesting way of doing this. I would have expected the text_updated_at field to be set in the APIView method, but if that code works, it is good.